tests/modules/appointments/services/service.spec.ts

```diff
@@ -0,0 +1,459 @@
+// createAppointment.test.js
+import sinon from 'sinon';
+import * as AppointmentRepository from '@/modules/appointments/repository/appointment.repository';
+import {
+  createAppointment,
+  deleteAppointment,
+  getAppointment,
+  getUserCreatedAppointments,
+  updateAppointment
+} from '@/modules/appointments/services/appointment.service';
+import { Prisma } from '@prisma/client';
+import { AppError } from '@/common/exceptions/appError';
+import * as DateUtils from '@/utils/date';
+
+describe('createAppointment', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  test('it should call createAppointment with the correct payload', async () => {
```

Comment:
the test message should be in the form
`it should return/throw object/error with ... payload`